test/index.js

```diff
@@ -3,6 +3,7 @@
 import test from 'tape';
 import irecord from '../index.js';
 import rx from './rx';
+import stampit from './stampit';
```

Comment:
eslint is warning they are unused because of how we are importing the test files. We could just create a new index and do a standard commonjs2 require to load them up.